unit_tests/kernels/UnitTestFaceElemBasic.C

```diff
@@ -223,3 +225,30 @@ TEST_F(Hex8ElementWithBCFields, faceElemScalarOpen)
 
   faceElemAlg.execute();
 }
+
+TEST_F(Hex8ElementWithBCFields, momentumInflowElemKernel)
```

Comment:
Move this to `MomentumKernelHex8Mesh` for easy filtering.

Reply:
This probably should stay here.  It's just checking that the inflowelem kernel doesn't segfault essentially.  There's still some work we need to do on getting good unit tests for all these boundary condition kernels